controller/Security.php

```diff
@@ -39,8 +47,74 @@ public function jwks(): void
         $this->setResponse($response);
     }
 
+    #
+    # @TODO Test method, will be removed...
+    #
+    public function launch(): void
+    {
+        /** @var Lti1p3RegistrationRepository $registrationRepository */
+        $registrationRepository = $this->getServiceLocator()->get(Lti1p3RegistrationRepository::class);
```

Comment:
```suggestion
        $registrationRepository = $this->getLti1p3RegistrationRepository()
```


and then add this somewhere else

```
private function getLti1p3RegistrationRepository(): Lti1p3RegistrationRepository
{
    return $this->getServiceLocator()->get(Lti1p3RegistrationRepository::class);
}
```

Reply:
The method `launch` is just to help you guys to test. It will be removed as soon the PR is approved :)